core/vm/jump_table.go

```diff
@@ -70,7 +70,7 @@ type JumpTable [256]*operation
 // contantinople, istanbul, petersburg, berlin, london, shanghai, curie, and descartes instructions.
 func newDescartesInstructionSet() JumpTable {
 	instructionSet := newCurieInstructionSet()
-
+	enable1153(&instructionSet)
```

Comment:
Let's keep this PR open for now. We have not decided whether this should go into Curie or Descartes.